### PR TITLE
Add toaster to add to work

### PR DESCRIPTION
**Pull request by ValeryiaMIRON**
- Add toaster to button add to work
- Add  a message depending on the status change of one candidate or several

Снимок экрана 2021-12-04 в 1 30 28 PM

Снимок экрана 2021-12-04 в 1 30 15 PM

https://trello.com/c/jmtZHo33/192-fe-add-toaster-to-btn-add-to-work

### CODE REVIEW

File: src/pages/candidates/Candidates.jsx [hunk 1]
```diff
@@ -54,13 +56,20 @@ const Candidates = () => {
...
+  useToaster();
+
   const handleClick = (event) => {
     setAnchorEl(anchorEl ? null : event.currentTarget);
   };
 
   const dispatch = useDispatch();
 
   const listOfCandidates = useSelector((state) => state.candidates.candidates);
+  const st = useSelector((state) => state.candidates)
```
Comment (nsukhotskaya) [inline src/pages/candidates/Candidates.jsx:69]: `st` - bad naming.

File: src/pages/candidates/Candidates.jsx [hunk 2]
```diff
@@ -54,13 +56,20 @@ const Candidates = () => {
...
+
+  console.log(st)
```
Comment (nsukhotskaya) [inline src/pages/candidates/Candidates.jsx:71]: Please delete console log statement

File: src/pages/candidates/Candidates.jsx [hunk 3]
```diff
@@ -129,6 +143,16 @@ const Candidates = () => {
 
   return (
     <Box padding="1%" width="100%" height="100%">
+       {alertMessages &&
```
Comment (nsukhotskaya) [inline src/pages/candidates/Candidates.jsx:146]: alertMessages- for errors, warnings( smth wrong).

Think anout another name.

For example, successMeesage

File: src/pages/candidates/Candidates.jsx (hunk 2 above)
Comment (ValeryiaMIRON) [reply src/pages/candidates/Candidates.jsx]: done

File: src/pages/candidates/Candidates.jsx (hunk 1 above)
Comment (ValeryiaMIRON) [reply src/pages/candidates/Candidates.jsx]: done

**Review by nsukhotskaya — APPROVED** [review]

**Review by OstroumovAlex — APPROVED** [review]

**Pull request merged**